packages/wcasa:work/lib/client/work.js

```diff
@@ -70,6 +70,10 @@ var changeTabs = function (event) {
             });
         }, 250);
     }
+    //Scroll to top of work on change tabs
```

Comment:
We now use airbnb's style guide.  Spaces are required after slashes:

`// Scroll to...`